src/activities/redactor.js

```diff
@@ -154,7 +155,13 @@ export function ortb2TransmitRules(isAllowed = isActivityAllowed) {
       paths: ORTB_GEO_PATHS,
       applies: appliesWhenActivityDenied(ACTIVITY_TRANSMIT_PRECISE_GEO, isAllowed),
       get(val) {
-        return Math.round((val + Number.EPSILON) * 100) / 100;
+        switch (typeof val) {
+          case 'string':
```

Comment:
I think it's better to add another, separate, rule for ip4 and another for ipv6 fields. They have different ORTB paths so there's no need to inspect them to figure out which is which here.